live.html

```diff
@@ -0,0 +1,350 @@
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 col-md-offset-2">
+                    <h1 class="title">Break through to the best life.</h1>
+                    <p>Crossroads is a nation-wide church community built to help you break-through the grind, connect to God, discover your purpose, and leave your mark on the world.</p>
+                    <p>
+                        <a class="btn btn-lg btn-primary" ui-sref="livestream" href="/live/stream">Watch Now</a>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div data-stream-upcoming="show" class="jumbotron jumbotron-xl flush-bottom hide" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/streaming-anywhere-hero-off.jpg?w=1&amp;crop=faces&amp;fit=crop');">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 col-md-offset-2">
+                    <h1 class="title">Break through to the best life.</h1>
+                    <p>The next live-streamed service is about to start. Grab a coffee, gather your friends and hold tight.</p>
+                    <div class="push-ends text-center crds-countdown countdown countdown-border countdown-white">
+                        <ul class="countdown-timer list-inline" data-stream-live="hide">
+                            <li class="countdown-days days">03</li>
+                            <li class="countdown-hours hours">00</li>
+                            <li class="countdown-minutes minutes">42</li>
+                            <li class="countdown-seconds seconds">37</li>
+                        </ul>
+                    </div>
+                    <p class="small">Crossroads is a nation-wide church community built to help you break-through the grind, connect to God, discover your purpose, and leave your mark on the world.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div data-stream-off="show" class="jumbotron jumbotron-xl flush-bottom hide" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/streaming-anywhere-hero-off.jpg?w=1&amp;crop=faces&amp;fit=crop');">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 col-md-offset-2">
+                    <p class="small-heading text-uppercase">Ready to leave the daily grind? So are we.</p>
+                    <h1 class="title">Hi. Welcome to Crossroads Anywhere!</h1>
+                    <p>No matter what you believe, we built this for you - and the thousands like you - who want to live by a different set of rules.</p>
+                    <div id="recent-message-btn">
+                        <a href="{{ latest_message.url }}" class="btn btn-lg btn-primary push-bottom"> Watch This Week's Service </a>

```

Comment:
I lead you astray here too: I think this will need to be parsed out a little differently. Try this:
`href="/series/{{ latest_message.series_slug }}/{{ latest_message.slug }}`

Reply:
also not going to work as it's a relative link. This needs to actually go to the mediaint subdomain...